src/axom/sidre/core/MFEMSidreDataCollection.hpp

```diff
@@ -256,6 +254,37 @@ class MFEMSidreDataCollection : public mfem::DataCollection
                      const std::string& buffer_name,
                      IndexType offset);
 
+  /// Register a mfem::QuadratureFunction in the Sidre DataStore.
+  /** This method is a shortcut for the call
+       `RegisterQField(field_name, qf, field_name, 0)`.
+    */
+  virtual void RegisterQField(const std::string& field_name,
+                              mfem::QuadratureFunction* qf)
+  {
+    RegisterQField(field_name, qf, field_name, 0);
+  }
+
+  /// Register a mfem::QuadratureFunction in the Sidre DataStore.
+  /** The registration procedure is as follows:
+       - if (@a qf's data is NULL), error out since this means there's no quadrature
```

Comment:
Please improve the docs here. `qf`'s data is `NULL` does not exactly match the check in the code:

```cpp
  if(field_name.empty() || buffer_name.empty() || qf == NULL || qf->GetSpace() == NULL)
```
It seems to me that the situation is that `qf` is null or doesn't have an associated quadrature space.